src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java

```diff
@@ -191,4 +194,36 @@ public DescriptorImpl() {
         }
     }
 
+    @Issue("JENKINS-30206")
```

Comment:
If this is something you claim is currently failing, then you should add `@Ignore` so that the PR itself is mergeable.